__tests__/comments.test.js

```diff
@@ -0,0 +1,64 @@
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+    return seed(testData);
+  });
+  
+  afterAll(() => {
+    return db.end();
+  });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+    test("status 200 , responds with  an array of comments objects ", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toHaveLength(3);
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+                review_id: expect.any(Number)

```

Comment:
Because we're making a get request for comments with the same review id we can expect `review_id` to be `2` for all of them